lib/cli.rb

```diff
@@ -286,8 +289,7 @@ def self.parse_scopes(scope_string)
 
       task = CompareTask.new
       opts = {
-          show_all: options["show-all"],
-          no_pager: !options["pager"]
+          show_all: options["show-all"]
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.